holopy/scattering/__init__.py

```diff
@@ -39,5 +39,9 @@
 
 '''
 
-from . import theory
-from . import scatterer
+from . import scatterer, theory
+from .theory import Mie, Multisphere, dda
+from .scatterer import Sphere, Spheres, JanusSphere, Ellipsoid, Capsule, Cylinder, Bisphere
+from .scatterer import Scatterer, Scatterers, Indicators, sphere_builtin, Union, Difference, Intersection
+from .calculations import calc_holo, calc_field, calc_intensity, calc_cross_sections, calc_scat_matrix
+
```

Comment:
Thanks for finally extending this import list. I have been meaning to do that for a while and just somehow never got around to it

Is there a reason you are importing lowercase dda (the file) rather than DDA (the theory object)?

I would like to rework some of the dda stuff which may affect these imports, but we can always change it here later.